cla_frontend/assets-src/javascripts/app/js/controllers/alternative_help.js

```diff
@@ -59,6 +59,14 @@
             });
           }
 
+          $scope.getF2fDeepLink = function () {
+            if ($scope.personal_details && $scope.personal_details.postcode) {
+              var postcode = $window.encodeURIComponent($scope.personal_details.postcode);
+              return 'http://find-legal-advice.justice.gov.uk/search.php?searchtype=location&searchtext='+ postcode +'&searchbtn=';
```

Comment:
Just had a thought on this...the postcode will be tracked in GA as all outbound links are tracked in full. It will also make our number for find-legal-advice segmented. Are we okay with this?